src/services/DFGFactor.jl

```diff
@@ -135,7 +135,7 @@ Return `::Bool` on whether given factor `fc::Symbol` is a prior in factor graph
 """
 function isPrior(dfg::G, fc::Symbol)::Bool where G <: AbstractDFG
   fco = getFactor(dfg, fc)
-  getFactorType(fco) isa FunctorSingleton
+  getFactorType(fco) isa AbstractPrior || getFactorType(fco) isa FunctorSingleton
```

Comment:
We should just remember to remove this with the deprecation.

Reply:
sounds good, the deprecation warnings in DFG v0.9 should be sufficient.  Dropping old abstracts in DFG v0.10 is a good plan

Reply:
I'm setting up this PR - there are a bunch of places they're still used, we just need to make sure v.10 won't break everything too badly.